src/App.tsx

```diff
@@ -15,71 +15,85 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
```

Comment:
I see that lot's of students are refactoring functional component into a class component in this taks. 
I don't see it as a mistake, but more out of a curiosity question: why not to use FC?